src/main/java/frc/robot/commands/GrabberLifterCommand.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.commands;
+import frc.robot.subsystems.GrabberLifter;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+public class GrabberLifterCommand extends CommandBase {
+    private final GrabberLifter m_GrabberLifter;
+    private double Speed;
+  
+    public GrabberLifterCommand(double speed, GrabberLifter subsystem) {
+  
+      m_GrabberLifter = subsystem;
+      double Speed = speed;
+      
+  
+      //addRequirements(m_Drivetrain);
+  
+      // Use addRequirements() here to declare subsystem dependencies
+      
+    }
+  
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {
+  
+      m_GrabberLifter.lift(Speed);
+  
+  
+    }
+  
+    // Called every time the scheduler runs while the command is scheduled.
+    @Override
+    public void execute() {
+  
+  
+  
+  
+    }
+  
+    // Called once the command ends or is interrupted.
+    @Override
+    public void end(boolean interrupted) {
+  
+      m_GrabberLifter.lift(Speed);
+  
```

Comment:
don't need this here: already setting speed on init.
set `isFinished()` to `true` such that the command stops running immediately after setting speed.